src/run.ts

```diff
@@ -286,27 +289,29 @@ ${
     projectId: context.projectId,
     state: 'opened',
     sourceBranch: versionBranch,
-    target_branch: branch,
+    target_branch: mrTargetBranch,
     maxPages: 1,
     perPage: 1,
   })
   console.log(JSON.stringify(searchResult, null, 2))
   if (searchResult.length === 0) {
-    console.log('creating merge request')
+    console.log(
+      `Creating merge request from ${versionBranch} to ${mrTargetBranch}.`,
```

Comment:
If the two are same, there is no need to mention it or the message can be simpler.

Reply:
Assuming running on branch `dev`. Then this message has two variants:

_if there's no `INPUT_TARGET_BRANCH` set_ 

> Creating merge request from changeset-release/dev to dev

_if `INPUT_TARGET_BRANCH` set to `main`_ 


> Creating merge request from changeset-release/dev to main

In both cases, `versionBranch` and `mrTargetBranch` are not the same and it's an useful log to have